flink-java/src/main/java/org/apache/flink/api/java/DataSet.java

```diff
@@ -1193,21 +1196,46 @@ public long count() throws Exception {
 		operation.setInput(this);
 		return operation.createResult();
 	}
-	
+
 	// --------------------------------------------------------------------------------------------
 	//  Union
 	// --------------------------------------------------------------------------------------------
 
 	/**
 	 * Creates a union of this DataSet with an other DataSet. The other DataSet must be of the same data type.
-	 * 
+	 *
 	 * @param other The other DataSet which is unioned with the current DataSet.
 	 * @return The resulting DataSet.
 	 */
 	public UnionOperator<T> union(DataSet<T> other){
 		return new UnionOperator<>(this, other, Utils.getCallLocationName());
 	}
 
+	/**
+	* Creates a set minus of this DataSet with an other DataSet. The other DataSet must be of the same data type.
+	*
+	* @param other The other DataSet which is set minus with the current DataSet.
+	* @return The resulting DataSet.
+	*/
+	public CoGroupOperator<T, T, T> minus(DataSet<T> other){
```

Comment:
This issue is about adding `EXCEPT` to the Table. The DataSet API which is touched here is a rather low level API and we are quite careful about adding new operators. Therefore, changes to the DataSet API should go through a separate JIRA issue. Please move this code to `DataSetMinus` class and revert the changes to this file. You can open a JIRA issue to discuss adding a `minus` operator to the DataSet API. Thank you.